component-library/component-lib/src/lib/shared/breadcrumb/breadcrumb.component.ts

```diff
@@ -48,7 +96,7 @@ export class BreadcrumbComponent implements OnInit, OnChanges {
         })
       }
     }
-    this.createLinks();
+    this.separatorIcon.size = this.config.size;
   }
 
   /**
```

Comment:
This function is really good, it's worth extracting and making it a global function, I can see this being used for something like a navigation component.